webpack/assets/javascripts/foreman_tools.js

```diff
@@ -85,19 +85,38 @@ export function initTypeAheadSelect(input) {
 // handle table updates via turoblinks
 export function updateTable(element) {
   let uri = new URI(window.location.href);
-  let search, perPage;
+  let searchTerm, perPage, isSearchForm;
 
   if (element !== undefined) {
-    search = $(element).find('.autocomplete-input').val() || $('#search-form').find('.autocomplete-input').val();
-    if (search !== undefined) {
-      uri.setSearch('search', search.trim());
+    isSearchForm = (element.id === 'search-form');
+    if ((isSearchForm || element.id === 'per_page') && getQueryStringValue(uri, 'page') > 1) {
```

Comment:
why do we need to check if page > 1? if it is 1 setting it to 1 is a noop, and it makes the `getQueryStringValue` function redundant.

Reply:
I think you missed this comment.